### PR TITLE
feat: add run cached demo

### PR DESCRIPTION
**Pull request by jim-wang-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?
make run-cached-demo only works when you have run-demo before.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -218,3 +218,9 @@ run-demo:
 	$(MAKE) run-camera-simulator
 	@echo Running Object_detection gRPC pipeline
 	PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_1
+
+run-cached-demo:
+	@echo "Running camera simulator"
+	$(MAKE) run-camera-simulator
+	@echo Running Object_detection gRPC pipeline
+	PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_1
```
Comment (vli11) [inline Makefile:226]: why not use camera_0 as it is first video file to use in case user to put own media file?

Comment (jim-wang-intel) [reply Makefile:226]: currently camera0 is for barcode use case i believe

Comment (jim-wang-intel) [reply Makefile:226]: also, run-demo is using camera1: https://github.com/intel-retail/automated-self-checkout/pull/435/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L220

Comment (jim-wang-intel) [reply Makefile:226]: so we better not change it

**Review by brian-intel — APPROVED** [review]

**Pull request merged**